src/core/loader/get-registry.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { itemsRegistry, initRegistry } from "../../components/shop/item-registry.js";
+
+export const loadRegistry = (screen, config) => {
+    const registryKeys = getRegistryKeys(config);
+    if (!registryKeys.length) return;
+
+    registryKeys.forEach(key => {
+        loadToCache(screen, key);
+    });
```

Comment:
can be done in one line `registryKeys.forEach(key => loadToCache(screen, key));`

Reply:
Used the currying approach suggested by Nick in the comment above, per 672e4a9